core/src/main/scala/org/bitcoins/core/api/wallet/db/SpendingInfoDb.scala

```diff
@@ -151,6 +151,10 @@ sealed trait SpendingInfoDb extends DbRowAutoInc[SpendingInfoDb] {
   /** The TXID of the transaction this output was received in */
   def txid: DoubleSha256DigestBE
```

Comment:
would be probably easier to just do this so the construction is impossible

`def txid: DoubleSha256DigestBE = outPoint.txIdBE`

Reply:
See #4017 , i didn't want to provide a default implementation because it will just get override by sub classes and make things more confusing IMO